cloudfoundry/utils_clients.go

```diff
@@ -73,6 +73,22 @@ func UsersToIDs(users []ccv2.User) []interface{} {
 	return ids
 }
 
+func IsErrNotAuthorized(err error) bool {
+	if _, ok := err.(ccerror.ForbiddenError); ok {
+		return true
+	}
+	if httpErr, ok := err.(ccerror.RawHTTPStatusError); ok && httpErr.StatusCode == 403 {
```

Comment:
it is a duplicate of the condition below, could you remove one ?

Reply:
ah, this duplication existed as well `IsErrNotFound()` which I used as base, so fixed it there too.